oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticIndexDefinition.java

```diff
@@ -142,6 +142,11 @@ public boolean isAnalyzed(List<PropertyDefinition> propertyDefinitions) {
         return propertyDefinitions.stream().anyMatch(pd -> pd.analyzed || pd.fulltextEnabled());
     }
 
+    @Override
+    protected String getDefaultFunctionName() {
+        return "lucene";
```

Comment:
Why not "elastic"? If we use "lucene", then at least we have to document here why. But I think we should try to use "elastic".

Reply:
Added comment for this.